azure-functions/acbs-function/src/functions/acbs-amend-facility-loan-record.js

```diff
@@ -38,11 +38,22 @@ df.app.orchestration('acbs-amend-facility-loan-record', function* amendFacilityL
 
     const { facilityId, facility, amendments, fmr } = payload;
     const { amendment } = amendments;
+
     let facilityLoanRecordAmendments;
 
     // 1.1. Facility Loan Record (FLR) amendment mapping
     const flrMApped = mappings.facility.facilityLoanAmend(amendments, facility, fmr);
 
+    /**
+     * When facility amendment is applied on facility type `Loan`,
+     * with only `amount` attribute execution should be terminated.
```

Comment:
i think this would be clearer

```js
/**
 * When a facility amendment is applied on a `Loan` facility type,
 * if there is no `amount` attribute,
 * execution should be terminated.
 */
```

Reply:
Above is incorrect as it is only applied when only the `amount` attribute is present.
Above suggestion is incorrect as if there is `coverEndDate` attribute then FLR record should be mapped.

Reply:
I have reworded the documentation.